x/oracle/types/asset.go

```diff
@@ -6,16 +6,16 @@ import (
 
 // Denomination constants
 const (
-	MicroAtomDenom  = "uatom"
-	MicroUmeeDenom  = "uumee"
-	USDDenom        = "USD"
-	BlocksPerMinute = uint64(10)
-	BlocksPerHour   = BlocksPerMinute * 60
-	BlocksPerDay    = BlocksPerHour * 24
-	BlocksPerWeek   = BlocksPerDay * 7
-	BlocksPerMonth  = BlocksPerDay * 30
-	BlocksPerYear   = BlocksPerDay * 365
-	MicroUnit       = int64(1e6)
+	AtomDenom       string = "ATOM"
+	UmeeDenom       string = "UMEE"
```

Comment:
IDK how these are used yet in the module, but if they're used in regards to actually moving funds around, they need to be the "base-level" denoms, e.g. `uumee` and `uatom`.

Also, I doubt we need `AtomDenom` at all.